ui/src/experiments/list/search/SearchExperimentsPanel.js

```diff
@@ -6,16 +6,14 @@ import { SegmenterContextProvider } from "providers/segmenters/context";
 
 import SearchExperimentsFilters from "./SearchExperimentsFilters";
 
-import "./SearchExperimentsPanel.scss";
-
 const SearchExperimentsPanel = ({ onChange, onClose, projectId }) => {
   return (
     <SegmenterContextProvider projectId={projectId}>
       <EuiFlyout
         id="experiments-search-panel"
-        className="searchPanelFlyout--left"
+        side="left"
         onClose={onClose}
-        size="s"
+        size="313px"
```

Comment:
I was wondering if we should fix this value here or should we do that in the `appConfig` or an `.scss` file

Reply:
If it's a one-off setting, I think its ok to be embedded here.